domino_data/auth.py

```diff
@@ -44,6 +73,15 @@ def get_headers(self) -> Dict[str, str]:
         if self.api_key:
             self.headers["X-Domino-Api-Key"] = self.api_key
 
+        if self.token_url is not None:
+            try:
+                jwt = get_jwt_token(self.token_url)
+            except httpx.HTTPStatusError:
+                pass
+            else:
+                self.headers["X-Domino-Jwt"] = jwt
+                return self.headers
```

Comment:
for my learning why are we returning different things in these two get headers?

Reply:
Proxy client is for the datasource proxy. We expect a different header than Domino because we don't do authentication at the proxy level (we use the JWT to get Domino user) and I preferred to use a header that conveyed that (similarly to API key header) instead of the traditional `Authorization` header used by actual authorization frameworks.